frontend/packages/console-shared/src/selectors/namespace.ts

```diff
@@ -3,3 +3,5 @@ import { K8sResourceKind } from '@console/internal/module/k8s';
 
 export const getRequester = (obj: K8sResourceKind): string =>
   _.get(obj, ['metadata', 'annotations', 'openshift.io/requester']);
+export const getDescription = (obj: K8sResourceKind): string =>
+  _.get(obj, ['metadata', 'annotations', 'openshift.io/description']);
```

Comment:
Can you switch these to optional chaining (`?.`) while in the code? It's better for type safety.

Reply:
Done.